lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -822,15 +823,14 @@ class HTMLInputElementImpl extends HTMLElementImpl {
 
   // https://html.spec.whatwg.org/multipage/input.html#concept-input-min-zero
   get _stepBase() {
-    const parseAttribute = attributeName => this._convertStringToNumber(this.getAttributeNS(null, attributeName));
     if (this.hasAttributeNS(null, "min")) {
-      const min = parseAttribute("min");
+      const min = ths._convertStringToNumber(this.getAttributeNS(null, "min"));
```

Comment:
Looks like the merge went wrong here; probably these three lines should not be changed?

Reply:
I did want to inline, as the code itself is easy enough to not really need the extra function in there, but mispelled `this`, in the process, should be corrected now.